common/lib/xmodule/xmodule/lti_module.py

```diff
@@ -449,11 +514,15 @@ def get_context(self):
             'accept_grades_past_due': self.accept_grades_past_due,
         }
 
-    def get_html(self):
+    def student_view(self, context):
```

Comment:
Instead of `get_html()` from the [`XModule`](https://github.com/edx/edx-platform/blob/b13f207e1f5ec313998d08a106d7c0d2d4eee6c6/common/lib/xmodule/xmodule/x_module.py#L1014-L1023), specify the full `student_view()`.